src/molecule/command/base.py

```diff
@@ -90,6 +91,8 @@ def execute_cmdline_scenarios(scenario_name, args, command_args, ansible_args=()
     :returns: None
 
     """
+    prepare_environment()
```

Comment:
I've not checked but will this properly fill ansible-lint options ? If someone is having:
```
skip_list:
- 'role-name'
```
Will this be properly handled ?

Reply:
Yep. I do have a WIP change on my fork of your repo, I used it as a model. That is how I had to make two more patches for the linter :P